src/complexity_measures/missing_dispersion.jl

```diff
@@ -6,31 +6,19 @@ export MissingDispersionPatterns
 
 """
     MissingDispersionPatterns <: ComplexityEstimator
-    MissingDispersionPatterns(est = Dispersion())
+    MissingDispersionPatterns(est = Dispersion()) → mdp
 
-An estimator for the number of missing dispersion patterns (``N_{MDP}``), a complexity
+An estimator for the number of missing dispersion patterns (MDP), a complexity
 measure which can be used to detect nonlinearity in time series [Zhou2023](@cite).
 
 Used with [`complexity`](@ref) or [`complexity_normalized`](@ref), whose implementation
 uses [`missing_outcomes`](@ref).
 
 ## Description
 
-If used with [`complexity`](@ref), ``N_{MDP}`` is computed by first symbolising each
-`xᵢ ∈ x`, then embedding the resulting symbol sequence using the dispersion pattern
-estimator `est`, and computing the quantity
-
-```math
-N_{MDP} = L - N_{ODP},
-```
-
-where `L = total_outcomes(est)` (i.e. the total number of possible dispersion patterns),
-and ``N_{ODP}`` is defined as the number of *occurring* dispersion patterns.
-
-If used with [`complexity_normalized`](@ref), then ``N_{MDP}^N = (L - N_{ODP})/L`` is
-computed. The authors recommend that
-`total_outcomes(est.symbolization)^est.m << length(x) - est.m*est.τ + 1` to avoid
-undersampling.
+When used with [`complexity`](@ref), `complexity(mdp)` is syntactically equivalent
+with just [`missing_outcomes`](@ref)`(est)`. When used with [`complexity_normalized`](@ref),
+we further divide `missing_outcomes(est)/total_outcomes(est)`.
```

Comment:
```suggestion
the normalization is simply `missing_outcomes(o)/total_outcomes(o)`.
```

`o` vs `est`: Outcome space, not probabilities estimator